besspin/cyberPhys/ui/hacker-kiosk/src/background.js

```diff
@@ -25,6 +27,31 @@ ipcMain.on('can-poll', (event) => {
   }
 })
 
+// ZMQ Network
+let zmq_sock = zmq.socket("pair");
+let zmq_address = "tcp://127.0.0.1:3333";
+zmq_sock.connect(zmq_address);
+
+let zmqQueue = [];
+
+zmq_sock.on('message', (msg) => {
+  let decoded = JSON.parse(msg);
+  console.log("zmq message recieved: ", decoded);
+  zmqQueue.push(decoded);
+});
+
+ipcMain.on('zmq-poll', (event) => {
+  event.reply('zmq-results',  JSON.parse(JSON.stringify(zmqQueue)));
+  zmqQueue = [];
+});
+
+ipcMain.on('cmd-msg', (event, cmd_id) => {
```

Comment:
I think this needs to be updated to reflect the JSON exchange format (see my comment below).